pyrogram/methods/messages/send_dice.py

```diff
@@ -47,8 +47,9 @@ async def send_dice(
                 For a contact that exists in your Telegram address book you can use his phone number (str).
 
             emoji (``str``, *optional*):
-                Emoji on which the dice throw animation is based. Currently, must be one of "🎲",  "🎯", "🏀" or "⚽️".
-                Defaults to "🎲".
+                Emoji on which the dice throw animation is based.
+                Currently, must be one of :obj:`~pyrogram.emoji.SOCCER_BALL`,  :obj:`~pyrogram.emoji.BASKETBALL`, :obj:`~pyrogram.emoji.DIRECT_HIT` or :obj:`~pyrogram.emoji.GAME_DIE`.
```

Comment:
Emoji references don't seem to play nice with docs. We'd better revert to what it was for now.